projects/storefrontapp-e2e-cypress/cypress.ci.json

```diff
@@ -11,7 +11,7 @@
   },
   "videoUploadOnPasses": false,
   "env": {
-    "API_URL": "https://20.55.1.150:9002",
+    "API_URL": "https://spartacus-demo.eastus.cloudapp.azure.com:8443",
```

Comment:
Is this change intentional?

Reply:
We need to revert the api url to what is was before in all the files of this PR.